### ISSUE TITLE
Hang with unusual content in textarea

### ISSUE THREAD

**Issue by cdot** [tinymce/tinymce#3555]
Certain strings in a textarea cause a hang (in Chromium 56.0.2924.76 and Firefox 52.0.1, at least). I have narrowed it down to a tiny example
[crispyduck.txt](https://github.com/tinymce/tinymce/files/867781/crispyduck.txt)
.

**Comment by donShakespeare** [comment]
Confirmed!

**Comment by fyrkant** [comment]
Connected to #3300 #3047 #3454

**Comment by lnewson** [comment]
Hi,

I was able to reproduce this in TinyMCE 4.5.11, however I'm unable to reproduce it in the latest version of TinyMCE. Here's a fiddle to show it working: http://fiddle.tinymce.com/Pghaab

It's likely this has been fixed somewhere between 4.5.11 and 4.9.10, so I'm going to close this issue. If you're still able to reproduce this in the latest version however, then let us know and we'll reopen this issue.

Thanks!
